crates/bevy_ecs/src/world/unsafe_world_cell.rs

```diff
@@ -368,6 +368,24 @@ impl<'w> UnsafeWorldCell<'w> {
         Ok(UnsafeEntityCell::new(self, entity, location))
     }
 
+    /// Retrieves an [`UnsafeEntityCell`] that exposes read and write operations for the given `entity`.
+    /// Similar to the [`UnsafeWorldCell`], you are in charge of making sure that no aliasing rules are violated.
+    #[inline]
+    pub(crate) fn get_entity_with_ticks(
```

Comment:
Do you want to make this `pub` so that it's possible for third-party callers to set the ticks correctly if they want?  

```suggestion
    pub fn get_entity_with_ticks(
```

Reply:
Sure!